### PR TITLE
modify builder.md and run.md of /doc/reference/

### PR DESCRIPTION
**Pull request by CandiceGuo**
**- What I did**
I have modified several English grammar mistakes in two files. They are  /doc/reference/builder.md and /doc/reference/run.md. 
**- How I did it**
Errors always occur on the third person singular.
**- How to verify it**
I found them and corrected them.
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/builder.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ that uses the current directory as context:
     ...
 
 The build is run by the Docker daemon, not by the CLI. The first thing a build
-process does is send the entire context (recursively) to the daemon.  In most
+process does is to send the entire context (recursively) to the daemon.  In most
```
Comment (allencloud) [inline docs/reference/builder.md]: I think this part is reasonable. `to` is not necessary.

**Comment by CandiceGuo** [comment]
How to solve the problem of dco-signed?

**Comment by CandiceGuo** [comment]
I  fixed my key issues and forced pushed. But it seems the dco/no tag not remove. How can I do ?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Your fork was probably out of date when rebasing, and because of that, some unrelated commits ended up in your branch. I also see that you opened your pull request from your "master" branch; it's best to create a separate "feature" branch for each pull request so that you can work on them without interfering with changes to master. (see https://docs.docker.com/opensource/workflow/find-an-issue/#/sync-your-fork-and-create-a-new-branch)

To fix this;

If your "upstream" (docker) remote is called "upstream", this should do it;

```
git fetch upstream; git fetch origin; git rebase upstream/master;
```

after that, _force_ push to your own fork;

```
git push -f origin
```

More information can also be found in our contributors guide; https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently

File: builder/tarsum.go [hunk 2]
```diff
@@ -104,7 +104,8 @@ func MakeTarSumContext(tarStream io.Reader) (ModifiableContext, error) {
 		return nil, err
 	}
 
-	if err := chrootarchive.Untar(sum, root, nil); err != nil {
+	err = chrootarchive.Untar(sum, root, nil)
+	if err != nil {
```
Comment (allencloud) [inline builder/tarsum.go]: I think it is not necessary to make such kind of change.

File: docs/reference/builder.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ Before the Docker daemon runs the instructions in the `Dockerfile`, it performs
 a preliminary validation of the `Dockerfile` and returns an error if the syntax is incorrect:
 
     $ docker build -t test/myapp .
-    Sending build context to Docker daemon 2.048 kB
+    Sending build context to Docker daemons 2.048 kB
```
Comment (thaJeztah) [inline docs/reference/builder.md]: Sorry, this is not correct, `daemon` is singular (there's only one daemon this is being sent to)

File: docs/reference/builder.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ the `Using cache` message in the console output.
 
 Build cache is only used from images that have a local parent chain. This means
 that these images were created by previous builds or the whole chain of images
-was loaded with `docker load`. If you wish to use build cache of a specific
+were loaded with `docker load`. If you wish to use build cache of a specific
```
Comment (thaJeztah) [inline docs/reference/builder.md]: this is not correct either; "the whole chain of images" is also singular (the _chain_ (of images) was loaded with `docker load`)

**Review by thaJeztah — CHANGES_REQUESTED** [review]
@CandiceGuo thanks for contributing, but unfortunately these changes are not correct (see my review inline).

Also, I see there are still some "foreign" commits in your pull request that shouldn't be in there.

I recommend reading the contributors guide (see my earlier comment for links) to get familiar with the workflow.

I'll close this pull request, but welcome seeing future contributions from your hand!

**Pull request closed without merging**